pkgs/development/libraries/libtiff/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, pkgconfig, zlib, libjpeg, xz }:
+{ stdenv, fetchurl, fetchpatch, pkgconfig, zlib, libjpeg, xz }:
```

Comment:
Not super new. It basically normalizes then hashes patches to account for possible variations in patch autogeneration processes, like from GitHub.